Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -1370,6 +2358,53 @@ def get_project_team(project_name):
         return project_team
 
 
+def copy_to_forensics_account(resource_id):
+    """
+    Copy resource Volumes to a Forensics Account
+    """
+    demisto.info(f"resource id is {resource_id}\n")
+    demisto.debug("copy_to_forensics_account, enter")
+
+    if not is_valid_uuid(resource_id):
+        variables = {
+            "first": 1,
+            "filterBy": {
+                "providerUniqueId": [resource_id]
+            }
+        }
+        resource_id_response = checkAPIerrors(PULL_RESOURCES_ID_NATIVE_QUERY, variables)
+        if resource_id_response['data'] is None or not resource_id_response['data']['cloudResources']['nodes']:
+            demisto.error(f"Resource with ID {resource_id} not found.")
+            return f"Resource with ID {resource_id} not found."
+        else:
+            resource_id = resource_id_response['data']['cloudResources']['nodes'][0]['id']
+
+    copy_to_forensics_account_mutation = """
```

Comment:
you don't want to move this mutation on the top of the script with all the queries/mutations?